common/siklu/siklu_board_ctu_ipq6018.c

```diff
@@ -1,13 +1,29 @@
+#include <asm/io.h>
 #include <asm/arch-qca-common/gpio.h>
 #include <siklu/siklu_board_ctu_ipq6018.h>
 
-#define POWER_LED_YELLOW_COLOR_GPIO_PIN 23
-
+/*
+ * The green color on the power LED is connected to GPIO 24. GPIO 24 is muxed
+ * to the PWM2 function. Configure PWM2 to 1 Hz with 50% duty cycle to make the
```

Comment:
Maybe better to match the same blink rate as the n366?

Reply:
OK. I'll change the blink rate to be the same.